core/src/test/java/nu/yona/server/subscriptions/service/PrivateUserDataMigrationServiceIntegrationTest.java

```diff
@@ -115,32 +90,24 @@ public void upgrade(User userEntity)
 	private final String password = "password";
 	private User richard;
 
-	@BeforeClass
-	public static void setUpForAll()
-	{
-		LocaleContextHolder.setLocale(Translator.EN_US_LOCALE);
-	}
-
 	@Before
 	public void setUpPerTest()
 	{
-		userAnonymizedRepository.deleteAll();
-		userRepository.deleteAll();
-		JUnitUtil.setUpRepositoryMock(mockMessageSourceRepository);
-
-		Map<Class<?>, Repository<?, ?>> repositoriesMap = new HashMap<>();
-		repositoriesMap.put(User.class, userRepository);
-		repositoriesMap.put(MessageSource.class, mockMessageSourceRepository);
-		repositoriesMap.put(UserAnonymized.class, userAnonymizedRepository);
-		JUnitUtil.setUpRepositoryProviderMock(repositoriesMap);
-
 		try (CryptoSession cryptoSession = CryptoSession.start(password))
 		{
 			richard = JUnitUtil.createRichard();
 		}
 		reset(userRepository);
 	}
 
+	@Override
+	protected Map<Class<?>, Repository<?, ?>> getRepositories()
+	{
+		Map<Class<?>, Repository<?, ?>> repositoriesMap = new HashMap<>();
+		repositoriesMap.put(MessageSource.class, mockMessageSourceRepository);
```

Comment:
Why are the user repositories not included here as they are in `BuddyDeviceChangeMessageDtoTest`?

Reply:
It's the opposite: why are they included in ``BuddyDeviceChangeMessageDtoTest`` and ``AddFirstDeviceTest``? And the answer to that is: for no good reason.

I'll push an update that removes them there.